demonstrations/vqe_parallel.py

```diff
@@ -52,44 +52,23 @@
 # hydrogen. This is achieved by finding the ground state energy of :math:`H_{2}` as we increase
 # the bond length between the hydrogen atoms.
 #
-# Each inter-atomic distance results in a different qubit Hamiltonian. To find the corresponding
-# Hamiltonian, we use the :func:`~.pennylane.qchem.molecular_hamiltonian` function of the
-# :mod:`~.pennylane.qchem` package. Further details on the mapping from the electronic
-# Hamiltonian of a molecule to a qubit Hamiltonian can be found in the
+# Each inter-atomic distance results in a different qubit Hamiltonian. Further
+# details on the mapping from the electronic Hamiltonian of a molecule to a
+# qubit Hamiltonian can be found in the
 # :doc:`tutorial_quantum_chemistry` and :doc:`tutorial_vqe`
 # tutorials.
 #
-# We begin by creating a dictionary containing a selection of bond lengths and corresponding data
-# files saved in `XYZ <https://en.wikipedia.org/wiki/XYZ_file_format>`__ format. These files
-# follow a standard format for specifying the geometry of a molecule and can be downloaded as a
-# Zip from :download:`here <../demonstrations/vqe_parallel/vqe_parallel.zip>`.
-
-data = {  # keys: atomic separations (in Angstroms), values: corresponding files
-    0.3: "vqe_parallel/h2_0.30.xyz",
-    0.5: "vqe_parallel/h2_0.50.xyz",
-    0.7: "vqe_parallel/h2_0.70.xyz",
-    0.9: "vqe_parallel/h2_0.90.xyz",
-    1.1: "vqe_parallel/h2_1.10.xyz",
-    1.3: "vqe_parallel/h2_1.30.xyz",
-    1.5: "vqe_parallel/h2_1.50.xyz",
-    1.7: "vqe_parallel/h2_1.70.xyz",
-    1.9: "vqe_parallel/h2_1.90.xyz",
-    2.1: "vqe_parallel/h2_2.10.xyz",
-}
-
-##############################################################################
-# The next step is to create the qubit Hamiltonians for each value of the inter-atomic distance.
-# We do this by first reading the molecular geometry from the external file using the
-# :func:`~.pennylane.qchem.read_structure` function and passing the atomic symbols
-# and coordinates to :func:`~.pennylane.qchem.molecular_hamiltonian`.
+# We begin by downloading a selection of Hamiltonians for :math:`H_2` for
+# various bond lengths using the
+# `PennyLane Datasets library <https://pennylane.ai/datasets/qchem/h2-molecule>`__:
 
+bonds = [0.5, 0.58, 0.7, 0.9, 1.1, 1.3, 1.5, 1.7, 1.9, 2.1]
+dataset = qml.data.load("qchem", molname="H2", bondlength=bonds, basis="STO-3G")

```

Comment:
Consider using `datasets`. 
```suggestion
datasets = qml.data.load("qchem", molname="H2", bondlength=bonds, basis="STO-3G")
```